ci/terraform/oidc/build-overrides.tfvars

```diff
@@ -11,6 +11,7 @@ spot_enabled                        = false
 language_cy_enabled                 = true
 internal_sector_uri                 = "https://identity.build.account.gov.uk"
 extended_feature_flags_enabled      = true
+support_auth_orch_split             = true
```

Comment:
I know I suggested using the same flag, so apologies if this is confusing. Is it important to have this turned on in Build now? The reason I ask is because the feature flag changes the behaviour of the authorize endpoint in a way that would break the acceptance tests right now; so if you do need it toggled on in Build, I guess maybe it does need a different flag after all.